README.md

```diff
@@ -1,79 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
+* Saves a log file of your experiment job output.
+* Notifies you when a long-running experiment job has finished.
+* Can be viewed remotely via both a web and mobile app.
 
-## Usage
+Hyperdash is compatible with: **Python 2.7-3.6**
 
-The Hyperdash SDK requires a valid API key in order to function. Luckily, the `hyperdash login` (if you already have an account) and `hyperdash signup` (if you don't) commands will automatically install one for you.
+## Command Line Installation
+*Foreword: We care deeply about making Hyperdash fast and easy to install  on Linux, Mac, and Windows. If you find a snag along the way, please let us know at support@hyperdash.io!*
 
-If you'd rather manage your API key manually, then review the "API Key Storage" section below.
+Install Hyperdash from [pip](https://packaging.python.org/tutorials/installing-packages/#requirements-for-installing-packages):
+```bash
+$ pip install --upgrade pip && pip install hyperdash
+```
+Installing within a python virtual environment such as [virtualenv](https://github.com/pypa/virtualenv) or [conda](https://github.com/conda/conda) is recommended.
+```bash
+# Login if you have an account
+$ hyperdash login
 
-### Standalone Script
-The easiest way to use the Hyperdash SDK is to simply prefix any terminal command with `hyperdash run`:
+# Or signup free with an email
+$ hyperdash signup
+```
+After `login` or `signup`, an API key is saved to your local machine for automatic authorization. If you'd rather manage your API key manually, then review the "API Key Storage" section below.
 
+You're ready to use Hyperdash! Make sure Hyperdash works by running:
 ```
-hyperdash run -n "My test python script" python my_test_script.py
+$ hyperdash demo
```

Comment:
Might as well update the script in the demo command so that it generates hyperparameters and metrics

Reply:
Agreed.